iquip/apps/visualizer.py

```diff
@@ -0,0 +1,142 @@
+"""App module for showing the code and sequence viewer."""
+
+import logging
+from typing import Callable, Optional, Tuple
+
+import requests
+from PyQt5.QtCore import QObject, Qt, QThread, pyqtSignal
+from PyQt5.QtWidgets import QLabel, QVBoxLayout, QWidget
+from PyQt5.Qsci import QsciScintilla, QsciLexerPython
+
+import qiwis
+
+logger = logging.getLogger(__name__)
+
+
+class CodeViewerFrame(QWidget):
+    """Frame for showing the code.
+    
+    Attributes:
+        editor: The QsciScintilla widget for showing the experiment code.
+    """
+
+    def __init__(self, parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(parent=parent)
+        # widgets
+        self.editor = QsciScintilla(self)
+        lexer = QsciLexerPython(self.editor)
+        self.editor.setFolding(True)
+        self.editor.setLexer(lexer)
```

Comment:
If it is not difficult, I would like to set the font as monospaced (fixed-width).